ui/raidboss/data/03-hw/raid/a8s.js

```diff
@@ -561,4 +561,131 @@
       },
     },
   ],
+  timelineReplace: [
+    {
+      'locale': 'cn',
+      'replaceSync': {
+        'Vortexer': '环旋者',
+        'Swindler': '欺诈者',
+        'Steam Regulator B': '蒸汽调整者β',
+        'Steam Regulator A': '蒸汽调整者α',
+        'Steam Gasket': '蒸汽垫',
+        'Power Plasma Gamma': '强离子体γ',
+        'Power Plasma Beta': '强离子体β',
+        'Onslaughter': '突击者',
+        'J Storm': '正义旋风',
+        'Hidden Mine': '隐形地雷',
+        'Brute Justice': '残暴正义号',
+        '(?! )Brawler(?<! )': '争斗者',
+        'Blaster Mirage': '爆破者幻象',
+        '(?<! )Blaster(?! )': '爆破者',
+        'Eye Of The Chakram': '战轮光束',
+        'Steam Chakram': '蒸汽战轮',
+        'Reset-': '重置',
+        'Added new combatant Blaster': '增加新的爆破者',
+      },
+      'replaceText': {
+        'attack': '攻击',
+        'Verdict': '终审开庭',
+        'Unknown Ability': 'Unknown Ability',
+        'Ultra Power Plasma': '究极强离子',
+        'Ultra Flash': '究极闪光',
+        'Transform': '正义合体',
+        'Supercharge': '超突击',
+        'Super Jump': '超级跳跃',
+        'Super Cyclone': '超级气旋',
+        'Single Buster': '破坏炮击',
+        'Short Needle': '小型导弹',
+        'Self-detonate': '自爆',
+        'Seed of the Sky': '天空之种',
+        'Rocket Drill': '火箭飞钻',
+        'Power Tackle': '强力前冲拳',
+        'Power Plasma': '强离子',
+        'Perpetual Ray': '永恒射线',
+        'Missile Command': '导弹齐发',
+        'Mirage': '幻影系统',
+        '(?<! )Minefield(?! )': '地雷散布',
+        'Mind Blast': '精神冲击',
+        'Mega Beam': '巨型光束炮',
+        'Magicked Mark': '魔力射击',
+        'Long Needle': '大型导弹',
+        'Legislation': '法制整顿',
+        'Justice': '正义合神',
+        'J Kick': '正义飞踢',
+        'Ice Missile': '寒冰导弹',
+        'Hydrothermal Missile': '蒸汽导弹',
+        'Hidden Minefield': '隐形地雷散布',
+        'Hidden Mine(?!ield)': '地雷爆炸',
+        'Height': '高度算术',
+        'Gavel': '终审闭庭',
+        'Gale Force': '暴风',
+        'Flarethrower': '大火炎放射',
+        'Final Punch': '终极双重火箭飞拳',
+        'Final Beam': '终极巨型光束炮',
+        'Final Apocalypse': '终极末世宣言',
+        'Eye of the Chakram': '战轮光束',
+        'Explosion': '爆炸',
+        'Execution': '执行准备',
+        'Elemental Jammer': '元素干扰',
+        'Earth Missile': '大地导弹',
+        'Drill Drive': '钻头驱动',
+        'Double Rocket Punch': '双重火箭飞拳',
+        'Double Drill Crush': '双重飞钻冲击',
+        'Double Buster': '双重破坏炮击',
+        'Discoid': '圆盘',
+        'Discharge': '枪击',
+        'Crashing Wave': '冲击波',
+        'Brute Force': '残暴铁拳',
+        'Blinder': '混合导弹',
+        'Bio-arithmeticks': '生命计算术',
+        'Ballistic Missile': '导弹发射',
+        'Attachment': '配件更换',
+        'Apocalyptic Ray': '末世宣言',
+        '100-Megatonze Shock': '亿万吨震荡',
```

Comment:
`882:9  error  Duplicate key '100-Megatonze Shock'  no-dupe-keys`
I didn't learn about js, so I don't know detailed reasons.. but compare with o2s, skill name is `100 Gs` but auther uses
`'(?<!-)100 Gs': '100 G',`
`'(?<!-)100 Gs': '重力100',`
`'(?<!-)100 Gs': '중력 100',`
Maybe it can't read 100 so they find a way to make a detour..?